airflow/providers/cncf/kubernetes/operators/spark_kubernetes.py

```diff
@@ -88,6 +89,7 @@ def __init__(
         reattach_on_restart: bool = True,
         delete_on_termination: bool = True,
         kubernetes_conn_id: str = "kubernetes_default",
+        override_existing: bool = True,
```

Comment:
The default behavior should stay the same:
```suggestion
        override_existing: bool = False,
```